src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -77,6 +79,10 @@ internal static string GetUniqueIdFromDiagnostic(Diagnostic diagnostic)
             "PSPossibleIncorrectUsageOfRedirectionOperator"
         };
 
+        private static readonly StringComparison s_osPathStringComparison = RuntimeInformation.IsOSPlatform(OSPlatform.Linux)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 's_osPathStringComparison'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5068680)